web_tree_dynamic_colored_field/static/src/js/web_tree_dynamic_colored_field.js

```diff
@@ -10,21 +10,37 @@ odoo.define('web_tree_dynamic_colored_field', function (require) {
          *
          * @override
          */
-        _renderBody: function () {
+        _renderView: function () {
             if (this.arch.attrs.colors) {
-                var colorAttr = this.arch.attrs.colors.split(';')
-                    .filter(color => color.trim().startsWith('color_field'));
+                var colors_attr = this.arch.attrs.colors.split(';')
+                var colorAttr = colors_attr.filter(
+                    attr => attr.trim().startsWith('color_field')
+                );
+                var backgroundColorAttr = colors_attr.filter(
+                    attr => attr.trim().startsWith('background_color_field')
```

Comment:
Arrow functions are not supported in IE
https://www.odoo.com/documentation/11.0/setup/deploy.html#supported-browsers
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions